### PR TITLE
feat(integrations): add support for dialpad sandbox

### PR DESCRIPTION
**Pull request by StephenCole19**
## Describe your changes

Added configuration for dialpad-sandbox in providers.yaml
Added dialpad-sandbox to docs-v2.
Added dialpad-sandbox pages for Supported API groups in mint.json.
Added dialpad-sandbox SVG file in template-logos folder.

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is: 

## Testing
Ran docker compose up.
Confirmed dialpad-sandbox existed.
Confirmed dialpad-sandbox worked with my sandbox client id.

### CODE REVIEW

**Review by bodinsamuel — APPROVED** [review]
Thanks for your contribution, lgtm 🙏🏻

**Comment by hassan254-prog** [comment]
Thank your @StephenCole19 , 

Nit: You could create a symbolic link to the template logo SVG file:
`ln -s dialpad.svg dialpad-sandbox.svg`

File: docs-v2/integrations/all/dialpad-sandbox.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Dialpad (Sandbox)
+sidebarTitle: Dialpad (Sandbox)
+---
+
+API configuration: [`dialpad-sandbox`](https://nango.dev/providers.yaml)
```
Comment (hassan254-prog) [inline docs-v2/integrations/all/dialpad-sandbox.mdx]: You can remove this doc page and have this line in the dialpad doc page. 
A good example would be the [Avalara](https://github.com/NangoHQ/nango/blob/a4ba67bcea027de7a60fda45b69232bc9e0ccd5c/docs-v2/integrations/all/avalara.mdx?plain=1#L6)

File: docs-v2/integrations/communication.mdx [hunk 2]
```diff
@@ -14,6 +14,7 @@ sidebarTitle: Communication
     <Card title="Chorus" href="/integrations/all/chorus" color="#68a063" />
     <Card title="Circle.so" href="/integrations/all/circle-so" color="#68a063" />
     <Card title="Dialpad" href="/integrations/all/dialpad" color="#68a063" />
+    <Card title="Dialpad (Sandbox)" href="/integrations/all/dialpad-sandbox" color="#68a063" />
```
Comment (hassan254-prog) [inline docs-v2/integrations/communication.mdx]: You can also omit this line.

File: docs-v2/mint.json [hunk 3]
```diff
@@ -372,6 +372,7 @@
                 "integrations/all/datadog",
                 "integrations/all/deel",
                 "integrations/all/dialpad",
+                "integrations/all/dialpad-sandbox",
```
Comment (hassan254-prog) [inline docs-v2/mint.json]: same as here since we now have a single docs page for both

**Comment by hassan254-prog** [comment]
You can try to consolidate the docs page by having small changes, I have commented inline. Other than that lgtm 👍.

**Comment by StephenCole19** [comment]
> You can try to consolidate the docs page by having small changes, I have commented inline. Other than that lgtm 👍.

@hassan254-prog Put everything in the one doc page :)

**Review by bodinsamuel — APPROVED** [review]

**Review by khaliqgant — APPROVED** [review]

**Review by hassan254-prog — APPROVED** [review]

**Pull request merged**